x-pack/plugins/triggers_actions_ui/public/application/sections/test_utils.tsx

```diff
@@ -52,15 +50,13 @@ export const createAppMockRenderer = (
   });
 
   const AppWrapper: React.FC<{ children: React.ReactElement }> = React.memo(({ children }) => (
-    <I18nProvider>
-      <KibanaRenderContextProvider {...core}>
-        <KibanaContextProvider services={services}>
-          <QueryClientProvider client={queryClient} context={queryClientContext}>
-            {children}
-          </QueryClientProvider>
-        </KibanaContextProvider>
-      </KibanaRenderContextProvider>
-    </I18nProvider>
+    <KibanaRenderContextProvider {...core}>
```

Comment:
This was always expected to work because `KibanaRenderContextProvider` wraps I18nProvider.